Source/WebCore/css/html.css

```
@@ -404,6 +404,10 @@ select, button, meter, progress, input:is([type="button"], [type="submit"], [typ
     writing-mode: horizontal-tb !important;
 }
 
+button, meter, progress, select {
+    user-select: none;
```

WebKit hasn't unprefixed user-select yet.

```suggestion
    -webkit-user-select: none;
```

---

Thanks I have modified the css.